### PR TITLE
version 2  tests 

### PR DESCRIPTION
**Pull request by keronei**
This forms the basis of the v2, the tests have been written before the actual API in order to shed some light to the actual writing of the endpoints.

### CODE REVIEW

File: app/tests/v2/test.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import unittest
+import json
+from ....app import launcher
+
+class TestEndPointsWithDatabases(unittest.TestCase):
+    def setUp(self):
+        self.app = launcher(app_config["development"])
+        self.client = self.app.test_client
+        #user_id should be autogenerated + date registered
```
Comment (thoth-ky) [inline app/tests/v2/test.py:9]: When you write a comment, you should start with a blank space (PEP8)
```suggestion
        # user_id should be autogenerated + date registered
```

**Review by thoth-ky — COMMENTED** [review]
Nice work with tests. Now add more edge cases and act on feedback left
